app/uk/gov/hmrc/residencenilratebandcalculator/controllers/ValueAvailableWhenPropertyChangedController.scala

```diff
@@ -72,20 +71,18 @@ class ValueAvailableWhenPropertyChangedController @Inject()(val messagesApi: Mes
       microserviceValues.map {
         case (nilRateValueJson, cacheMap) => {
           val previousAnswers = answerRows(cacheMap, request)
-          val userAnswers = new UserAnswers(cacheMap)
           val nilRateBand = CurrencyFormatter.format(nilRateValueJson.json.toString())
-          implicit val messages = messagesApi.preferred(request)
+          implicit val messages: Messages = messagesApi.preferred(request)
```

Comment:
This might be misunderstanding but shouldn't you be pulling this messagesApi from the DefaultMessagesControllerComponents (that is such a long name) that you pass in? Also if it's being injected in does this implicit messages need to be added?